dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -205,6 +205,50 @@ is equivalent to COM16:
 
 .. _using-sitl-for-ardupilot-testing_sitl_without_mavproxy_tcp:
 
+Replaying serial data from Saleae Logic data captures
+=====================================================
+
+Saleae Logic is often used when decoding and debugging protocols for the first time.  The "async serial" analyzers can export the data to a CSV, and this data can then be replayed through ArduPilot's simulated UARTDriver to test parsing of that data.
+
+Serial data is replayed into the simulation at the same rate it appeared on the wire when taking the trace, preserving frame-gaps and the like.
+
+After you have Logic decoding the serial stream, export the data using this interface element:
+
+.. figure:: ../images/saleae-async-save.png
```

Comment:
wrong location for where you put the image

Reply:
Weird.  I was sure I'd rendered this locally and it worked.

I've moved the image.